packages/cfd/src/Containers/cfd-password-modal.tsx

```diff
@@ -169,12 +170,20 @@ const ReviewMessageForMT5 = ({
         return (
             <Localize i18n_default_text='To start trading, top-up funds from your Deriv account into this account.' />
         );
-    } else if ([Jurisdiction.BVI, Jurisdiction.VANUATU].includes(jurisdiction_selected_shortcode)) {
+    } else if (
+        (jurisdiction_selected_shortcode === Jurisdiction.VANUATU ||
+            jurisdiction_selected_shortcode === Jurisdiction.BVI) &&
+        [Jurisdiction.BVI, Jurisdiction.VANUATU].includes(jurisdiction_selected_shortcode)
```

Comment:
@hamza-deriv why are you checking for both of these conditions?
`(jurisdiction_selected_shortcode === Jurisdiction.VANUATU ||
            jurisdiction_selected_shortcode === Jurisdiction.BVI)`
and
`[Jurisdiction.BVI, Jurisdiction.VANUATU].includes(jurisdiction_selected_shortcode)`

one of them is enough.

Reply:
The thing is juridisction_selected_shortcode contains all shortcode so TS gives error as others are not found. Thats why i added this check to get rid of this error. Lets see if i find any other solution for it.